pkg/controller/nodes/handler/state_test.go

```diff
@@ -0,0 +1,23 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/gob"
+	"testing"
+
+	"github.com/lyft/flytepropeller/pkg/controller/nodes/task/k8s"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDecodeTaskState(t *testing.T) {
```

Comment:
Can you add a comment on what this is testing and why/whether that str is meaningful?